lms/djangoapps/instructor/views/serializer.py

```diff
@@ -478,6 +478,23 @@ def validate_user(self, value):
         return user
 
 
+class CertificateGenerationSerializer(serializers.Serializer):
+    """Serializer for enabling or disabling self-generated certificates for a course."""
+    certificates_enabled = serializers.BooleanField(
+        source="certificates-enabled",
+        required=True,
```

Comment:
```suggestion
        required=False,
        default=False,
```

Reply:
This is a required field and cannot be empty, but we can use the default value.